nibabel/loadsave.py

```diff
@@ -105,25 +83,38 @@ def save(img, filename):
     -------
     None
     '''
+
+    # Save the type as expected
     try:
         img.to_filename(filename)
     except ImageFileError:
         pass
     else:
         return
-    froot, ext, trailing = splitext_addext(filename, ('.gz', '.bz2'))
+
+    # Be nice to users by making common implicit conversions
+    froot, ext, trailing = splitext_addext(filename, img._compressed_exts)
```

Comment:
Use of private `img._compressed_exts`?  Anyway - is that a good idea?  What if we are trying to save a `.mnc` image as `.nii.gz` for example?